SwiftReflector/Demangling/Swift5NodeToTLDefinition.cs

```diff
@@ -833,6 +834,40 @@ public TLDefinition Convert (Node node)
 			}
 		}
 
+
+		TLDefinition ConvertDispatchThunk (Node node)
+		{
+			switch (node.Children [0].Kind) {
+			case NodeKind.Getter:
```

Comment:
Question: why does this switch not need 'breaks'?

Reply:
Every includes a return.

Reply:
@tj-devel709 you will get a compiler error if is not correct: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs0163